warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -47,7 +49,8 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
                        url=None, ip=None, remote=None, element_config_file=None,
                        element_tag=None):
         """
-        This will launch a browser.
+        The Keyword would launch a browser and Navigate to the url, if provided by the user.
+        The keyword does not validate the url provided by the user.
```

Comment:
Hi @brettleehari,

Can line 53 be emphasized? Like it can be made all caps or have an underline?

Thanks.

Reply:
@sanikakulkarni I have underlined as per your comment. Is this Warrior convention ? @bjrr What is your view ?